Modules/_ctypes/callbacks.c

```diff
@@ -155,20 +160,17 @@ static void _CallPythonObject(void *mem,
 
     assert(PyTuple_Check(converters));
     nargs = PyTuple_GET_SIZE(converters);
-    /* Hm. What to return in case of error?
-       For COM, 0xFFFFFFFF seems better than 0.
-    */
     if (nargs < 0) {
         PrintError("BUG: PySequence_Length");
         goto Done;
     }
 
-    PyObject *args_stack[CTYPES_MAX_ARGCOUNT];
-    if (nargs <= CTYPES_MAX_ARGCOUNT) {
+    PyObject *args_stack[_PY_FASTCALL_SMALL_STACK];
```

Comment:
I don't see the point of always allocating this array on the stack if you use alloca(). Always use alloca(), no?

I suggest to avoid calling alloca(0): for example, set args to NULL if nargs==0.